src/main/drivers/pwm_stm32f4xx.c

```diff
@@ -31,6 +31,10 @@
 
 #define MAX_DMA_TIMERS 8
 
+#define MOTOR_BIT_0 17
```

Comment:
@borisbstyle nah - my maths was wrong ... the prescaler at when using 24mhz is going to return a fraction when on the slower bus (168000000/2/24000000)-1.

Will submit an update to run the timers at 12mhz and 3mhz.